body.tex

```diff
@@ -1,166 +1,464 @@
-for lsst-texmf. Once you have it installed, you should be able to compile your paper
-using make. 
+\poc's goal is to demonstrate that running Data Release Production (DRP) on AWS is possible.
+The following approaches were taken.
 
-\subsection{How to handle author list?} 
+\subsection{Progress in phases}
 
-Authors come from the authors.yaml file --  find the author ids in the lsst-texmf/etc/authordb.yaml - use db2authors to get the authors and institutes from the db. 
+\citeds{DMTN-114} proposed development phases that broadly outlined how progress can be made and when performance would be monitored. The outlined plan was to move the execution to AWS without any backend modifications first and then gradually switch out each component to AWS.
 
-{\bf XXX Wil, the above is unclear: need more detail about how to use db2authors,
-what is its output and what to do with it...} 
+Before this \poc, the DRP execution relied a shared POSIX-compliant filesystem.
+In the first phase such filesystem was replaced by Amazon EFS.
+The data repository, including a POSIX datastore and a SQLite registry, was entirely located on EFS that was mouted to each of the EC2 compute instances.
 
+Later, the S3 datastore replaced the need of EFS, and the PostgreSQL RDS replaced the SQLite registry.
+Also, a Personal HTCondor Pool on an single EC2 instance was used before we tested the capability to launch new EC2 instances as the HTCondor workers and execute jobs.
+This approach allowed us to debug and adapt more smoothly and always had a fallback option in integrating and testing new features and components.
 
-\subsection{How to handle LSST standard references?} 
+\subsection{Test minimally before scaling up}
 
-The papers should cite standard LSST references\footnote{See \url{https://github.com/lsst-pst/LSSTreferences}}, 
-where appropriate. For the usage, please see below.  These examples all use the ADS handle, unless they are 
-project docs then the use the project handle like LSE-17.
+Small tasks are tested before workflows of jobs, and small workflows are tested before large workflows.
+The canonical \href{https://github.com/lsst/ci\_hsc}{ci\_hsc} dataset and the accompanying CI workflow provide a minimal HSC test dataset and a representative DRP workflow and algorithms.
+This test workflow is an important step in integrating new changes.
+For larger scale tests we used the \href{https://jira.lsstcorp.org/browse/DM-11345}{HSC-RC2 dataset}.
 
-All are on the lsst-texmf which you can get from \url{http://lsst-texmf.lsst.io}
+\subsection{Use Generation 3 Middleware}
 
+One of the mandates in the \poc~was to use the Generation 3 Middleware (\citeds{DMTN-056}), which has been designed to ease the DRP execution and automation compared to the previous Generation 2 Middleware. Generation 3 Middleware design is more pliable to alternative backends, compared to Generation 2 Middleware.
```

Comment:
"... design is more readily extensible to alternative ..."

Reply:
Was fixed but left open. This made me re-write a sentence in the "Building AWS support into the Data Butler" section that also used pliable. Thanks!